### PR TITLE
Added Zapier.com

### PR DESCRIPTION
**Pull request by handrus**
Adding Zapier.com to the site list

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -7718,6 +7718,15 @@
             "yummly.com"
         ]
     },
+
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion

```
Remove trailing whitespaces

File: _data/sites.json [hunk 2]
```diff
@@ -7718,6 +7718,15 @@
             "yummly.com"
         ]
     },
+    
+    {
+        "name": "Zapier",
+        "url": "https://zapier.com/app/settings/delete",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json]: Why is it medium? Normally medium ones require additional steps, it would be good to describe those in the `notes` field.

**Comment by tupaschoal** [comment]
@handrus can you please clarify on the issue opened?

**Comment by tupaschoal** [comment]
Closing for lack of activity, please reopen if applicable

**Comment by handrus** [comment]
Sorry, completely missed notifications.
It is a medium difficulty because the site itself (at the time) didn't offer any direct button, link or hint to the address. I had to enter their forum, search for threads with this subject to find a answer that directed to the link.

**Pull request closed without merging**